docs/maintaining.md

```diff
@@ -11,14 +11,19 @@ They live in `bin/` and `_test/`.
 
 This is non-exhaustive.
 
-- Prefer snakecase for file names
-    ```sh
-    hello_world.sh
-    ```
-- Prefer BASH for scripting
 - Adopt a Unix philosophy for tooling
   - prefer using tools that do one thing well
   - prefer using tools that are ubiquitous: `jq` or `sed` instead of `prettier` or `sd`
   - write scripts to do one thing well
+- Prefer BASH for scripting
 - Scripts should use `#!/usr/bin/env bash` as their shebang
   - This increases portability on NixOS and macOS because contributors' preferred bash may not be installed in `/bin/bash`.
+- Prefer snakecase for script file names
+    ```sh
+    hello_world.sh
+    ```
+- Script file names should include the `.sh` extension
```

Comment:
Funny enough the shellcheck step warns against this: https://github.com/exercism/rust/pull/1159/checks?check_run_id=1937700463#step:3:7
So I may be misremembering the conclusion?

Reply:
In this case I agree with shellcheck. It's an accident of CI that that file is in the PATH. I'd prefer to suppress the warning rather than removing the file extensions.

Reply:
Well, I'm wrong. This is not a warning from shellcheck but the [GitHub Action](https://github.com/ludeeus/action-shellcheck/blob/d586102c117f97e63d7e3b56629d269efc9a7c60/action.yaml#L167).